com.playeveryware.eos/Documentation~/getting_started_guide/README.md

```diff
@@ -0,0 +1,235 @@
+<em>You can either sign in to an existing account, or create a new account, on the Epic Developer Portal.</em></p>
+
+After acquiring an account, if you are not using an existing Organization, you will be required to create an Organization.
+An Organization is a holder for all of the Products you make.
+
+See [Epic's documentation on Organizations](https://dev.epicgames.com/docs/dev-portal/organization-management) for more information.
+
+## Working With an Existing Organization
+
+If you or a team member has already created an Organization, you can add Epic Game Store accounts to that Organization.
+While signed in to the `Epic Games Developer Portal`, navigate to `Organization`.
+Under the `Members` tab, you can use the `Invite New` button to send an invitation email to the target user.
+That email will contain a link that will help the user sign up for an Epic Account if necessary, and then subsequently they will be added to your Organization.
+
+Everyone who will be testing your game should be a part of your Organization, or part of a Player Group. If they aren't, then whenever the EOS SDK requires the user to login, that user will be forbidden from accessing your application.
+
+For more information on Player Groups, see [Epic's documentation on Player Groups](https://dev.epicgames.com/docs/dev-portal/product-management#player-groups).
+
+# Developer Portal Configuration for Your Game
+
+Before bringing the EOS Plugin for Unity into your project, you must first configure your game in the Epic Developer Portal.
+This section will cover the creation and configuration of a `Product`, `Client Policy`, `Client`, and `Application`.
+During this process, the Developer Portal will create a set of `Sandbox` and `Deployment` environments for your game to use.
+
+> [!NOTE]
+> While navigating the Epic Developer Portal to certain features, there will be dialogues to review and accept Epic terms and services.
+> There are several of these throughout the onboarding process, with specific domains such as Epic Account Services. You will need to agree to the terms and services to use features that are part of those domains.
+> This will bring you to a Terms of Service page with detailed information about the agreement.
+> 
+> In order to utilize many features of Epic Online Services, you will need to review and agree to these conditions.
+> In order to use any features under `Epic Games Store`, there will be an agreement process that includes a possible submission fee.
+> While this agreement is required to perform Epic Game Store operations, including publishing and selling your game, the EOS Plugin for Unity can be configured and utilized before this is agreed to.
+>
+> Fore more information on the onboarding process, see [Epic's documentation on Get Started Overview](https://dev.epicgames.com/docs/epic-games-store/get-started/get-started-overview#onboarding-process).
```

Comment:
It should be 'for' not 'fore'.

Reply:
I ran it through a grammar/spelling check, and took its suggestions. I'm surprised how many things it caught! I'll do that moving forward when writing documentation.